modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+
+# Reference all required bash variables prior to running. Due to 'nounset', if
+# a caller fails to export the following expected environmental variables, this
+# script will fail immediately rather than partially succeeding.
+echo "Cloud SQL Instance Connection string: ${SQL_INSTANCE_CONN_STRING}"
+echo "SQL port: ${SQL_PORT}"
+echo "Forseti DB name: ${FORSETI_DB_NAME}"
+
+if ! [[ -f ${forseti_server_conf_path} ]]; then
+    echo "Could not find the configuration file: ${forseti_server_conf_path}." >&2
+    exit 1
+fi
+
+# We had issue creating DB user through deployment template, if the issue is
+# resolved in the future, we should create a forseti db user instead of using
+# root.
+SQL_SERVER_LOCAL_ADDRESS="mysql+pymysql://${SQL_DB_USER}:${SQL_DB_PASSWORD}@127.0.0.1:${SQL_PORT}"
+FORSETI_SERVICES="explain inventory model scanner notifier"
+
+FORSETI_COMMAND="$(which forseti_server) --endpoint '[::]:50051'"
+FORSETI_COMMAND+=" --forseti_db ${SQL_SERVER_LOCAL_ADDRESS}/${FORSETI_DB_NAME}?charset=utf8"
+FORSETI_COMMAND+=" --config_file_path ${forseti_server_conf_path}"
+FORSETI_COMMAND+=" --services ${FORSETI_SERVICES}"
+
+CONFIG_VALIDATOR_COMMAND="${forseti_home}/external-dependencies/config-validator/ConfigValidatorRPCServer"
+CONFIG_VALIDATOR_COMMAND+=" --policyPath='${policy_library_home}/policy-library/policies'"
+CONFIG_VALIDATOR_COMMAND+=" --policyLibraryPath='${policy_library_home}/policy-library/lib'"
+CONFIG_VALIDATOR_COMMAND+=" -port=50052"
+
+if [ "${policy_library_sync_enabled}" == "true" ]; then
+  POLICY_LIBRARY_SYNC_COMMAND="$(which docker) run -d"
+  POLICY_LIBRARY_SYNC_COMMAND+=" --log-driver=gcplogs"
+  POLICY_LIBRARY_SYNC_COMMAND+=" --log-opt gcp-log-cmd=true"
+  POLICY_LIBRARY_SYNC_COMMAND+=" --log-opt labels=git-sync"
+  POLICY_LIBRARY_SYNC_COMMAND+=" -v ${policy_library_home}:/tmp/git"
+  POLICY_LIBRARY_SYNC_COMMAND+=" -v /etc/git-secret:/etc/git-secret"
+  POLICY_LIBRARY_SYNC_COMMAND+=" k8s.gcr.io/git-sync:${policy_library_sync_git_sync_tag}"
+  POLICY_LIBRARY_SYNC_COMMAND+=" --branch=${POLICY_LIBRARY_REPOSITORY_BRANCH:=master}"
```

Comment:
I believe this is the error seen in the build. It looks like this is referencing an environment variable, so I think you can get rid of the curly brackets `$POLICY_LIBRARY_REPOSITORY_BRANCH:=master`
```
Step #2 - "converge":        Error: failed to render : <template_file>:55,77-78: Extra characters after interpolation expression; Expected a closing brace to end the interpolation expression, but found extra characters.
Step #2 - "converge":        
Step #2 - "converge":          on ../../../modules/server/main.tf line 118, in data "template_file" "forseti_server_initialize_services":
Step #2 - "converge":         118: data "template_file" "forseti_server_initialize_services" {
```

Reply:
Done